ci/runtests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source $HOME/miniconda/etc/profile.d/conda.sh
+conda activate test
+
+pip install -e .
+
+pytest -x -vv .
+
+if [ $? -ne 0 ]; then
+    exit 1
+fi
```

Comment:
This `if` is not necessary with the addition of `-ex` to the shebang, so please remove it (`-ex` makes a bash script to fail if any of the commands inside it fails).